inst/template/default.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang = "{{ lang }}">
```

Comment:
It would be a bit nicer if the whole `lang = "foo"` was conditional. I know that `lang=""` is valid, but it feels better to not have it at all in that case.